node-repository/src/test/java/com/yahoo/vespa/hosted/provision/provisioning/DynamicDockerProvisioningTest.java

```diff
@@ -60,7 +60,7 @@
      */
     @Test
     public void relocate_nodes_from_spare_hosts() {
-        ProvisioningTester tester = new ProvisioningTester(new Zone(Environment.prod, RegionName.from("us-east")), flavorsConfig());
+        ProvisioningTester tester = new ProvisioningTester.Builder().zone(new Zone(Environment.prod, RegionName.from("us-east"))).flavorsConfig(flavorsConfig()).build();
```

Comment:
Tester should be initialised in a `@Before` method, then each test can use that instead of repeating the code.

Reply:
Unfortunately, some of the tests in this class use different zones (prod vs. perf environment). I could initialize the prod tester, since its unused in most of the test, but I think it makes things less clear and then some helper methods would start referring to the instance variable rather being forced to take it in as argument...